cmd/kubeless/list.go

```diff
@@ -100,26 +100,33 @@ func doList(w io.Writer, tprClient rest.Interface, ns, output string, args []str
 func printFunctions(w io.Writer, functions []*spec.Function, output string) error {
 	if output == "" {
 		table := tablewriter.NewWriter(w)
-		table.SetHeader([]string{"Name", "namespace", "handler", "runtime", "type", "topic", "dependencies"})
+		table.SetHeader([]string{"Name", "description", "namespace", "handler", "runtime", "type", "topic", "dependencies", "memory", "env"})
 		for _, f := range functions {
 			n := f.Metadata.Name
+			desc := f.Spec.Desc
```

Comment:
@sebgoa Yeah It was there but then Gus suggested not storing the desc as an annotation or we should add kubeless-specific prefix, so I decided to put the description to the func spec.

> As far as I can tell, "Description" isn't one of the defined standard annotations. Why are you storing the description as an annotation, and why is the annotation key not prefixed with something kubeless-specific?